.concourse/rotate-ccdb-keys-test.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
```

Comment:
Maybe it's better for this file to not live under `.concourse/` so it is possible to run the test also outside of the ci context

Reply:
:+1: